addons/CLib/Core/Compression/fn_checkAllFunctionCompression.sqf

```diff
@@ -18,7 +18,7 @@
     private _fncName = _x;
     private _originalFunction = (parsingNamespace getVariable _fncName) call CFUNC(codeToString);
 
-    if (_originalFunction call CFUNC(checkCompression)) then {
+    if !(_originalFunction call CFUNC(checkCompression)) then {
```

Comment:
instead of using ! here why not just switch the contents around?

Reply:
No real reason... It's just what came into my mind first.
If you want, I can change it though

Reply:
its cleaner to read